Store.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Karma</title>
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/store.css">
+    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+</head>
+<body>
+
+    <!-- Add your HTML markup here -->
+    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <!-.- All the images you need are in the 'img' folder -->
+
+ 
+
+  <header>
+    
+        <img class="logo-img" src="img/karma-logo.svg" alt="karma-logo"/>
+          
+      
+      
+      <div>
+        <ul class='container'>
+          <li class='con-list'>
+            <a class='con-link' href="#">Meet Karma</a>
+          </li>
+          <li class='con-list'>
+            <a class='con-link' href="#">How it Works</a>
+          </li>
+          <li class='con-list'>
+            <a class='con-link' href="#">Store</a>
+          </li>
+          <li class='con-list'>
+            <a class='con-link' href="#">Blog</a>
+          </li>
+          <li class='con-list'>
+            <a class='con-link' href="#">Help</a>
+          </li>
+          <li class='con-list'>
+            <a class='con-link' href="#">Login</a>
+          </li>
+        </ul>
+    </div>
+
+  </header>
+  <main>
+      <section class="store-section">
+
+         <DIV>
+            <img class="store-img" src="img/store-image_by-andrew-neel-unsplash.jpg" alt=""/>
+
+        </DIV>
+        <DIV class="store-form">
+            <form>
+             <h1 class="form-header">Order your Karma wifi device today!</h1>
+                <fieldset class="row-form">
+                    <legend></legend>
+                    <div class="column-form spc">
+                        <label for="firstname">First name *</label>
+                    <input type="text" id="firstname" name="firstname" required>
+                    
+
+                    </div>
+                    <div class="column-form">
+                        <label for="lastname">Last name *</label>
+                        <input type="text" id="lastname" name="lastname" required>
```

Comment:
you can add some classes for your input, to make it look a little bit better( maybe make it a little bit smaller and change the width of it?)